docs/section10/10-7.md

```diff
@@ -17,120 +17,111 @@
                                                 
                 **Rule Assertion:**
-                - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or the modeled efficiency matches the most conservative efficiency across its size category and below and the efficiency was defined in the correct units then pass: `if efficiency_matches_acceptable_value == TRUE AND was_capacity_not_defined == FALSE: PASS`    
-                - Case 2: Elif capacity was not defined but the modeled efficiency meets the highest (most conservative) efficiency in the appropriate table based on system type then pass: `elif was_capacity_not_defined == TRUE AND capacity_not_defined_and_meets_most_conserv_eff_b ==  TRUE: PASS`  
-                - Case 3: Elif capacity was not defined and efficiency does not equal the highest (most conservative) efficiency in the appropriate table based on system type then UNDETERMINED: `elif was_capacity_not_defined == TRUE AND capacity_not_defined_and_meets_most_conserv_eff_b ==  FALSE: UNDETERMINED and raise_message "Check if the modeled baseline DX cooling efficiency was established correctly based upon equipment capacity and type."`          
-                - Case 4: Else, fail: `else: FAIL`  
+                - Case 1: if modeled efficiency equals the expected efficiency AND the capacity was defined AND the aggregation factor is either irrelevant or defined, then pass: `if modeled_efficiency_b == expected_baseline_eff_b AND not was_capacity_not_defined and zone_aggregation_factor_undefined_and_needed == FALSE: PASS`	
+                - Case 2: Elif capacity was not defined or and zone_aggregation_factor_undefined_and_needed equal TRUE but the modeled efficiency equals the highest (most conservative) efficiency in the appropriate table based on system type then UNDETERMINED: `elif (was_capacity_not_defined or zone_aggregation_factor_undefined_and_needed == TRUE) AND modeled_efficiency_b == most_conservartive_eff ==  TRUE: UNDETERMINED and raise message "Check if the modeled baseline DX cooling efficiency was established correctly based upon equipment capacity and type. The modeled efficiency matches the capacity bracket in Appendix G efficiency tables with the highest efficiency (i.e.,  most conservative efficiency has been modeled)."`   
+                - Case 3: Elif capacity was not defined or zone_aggregation_factor_undefined_and_needed equal TRUE and efficiency does not equal the highest (most conservative) efficiency in the appropriate table based on system type then UNDETERMINED: `elif (was_capacity_not_defined or zone_aggregation_factor_undefined_and_needed == TRUE) AND modeled_efficiency_b < most_conservartive_eff: UNDETERMINED and raise_message "Check if the modeled baseline DX cooling efficiency was established correctly based upon equipment capacity and type."`          
```

Comment:
again - suggest providing details so the user can make the required changes

Reply:
same reply as previous

Reply:
looks good